openmetadata-service/src/main/java/org/openmetadata/service/security/auth/BasicAuthenticator.java

```diff
@@ -111,6 +111,14 @@ public void init(OpenMetadataApplicationConfig config) {
     this.isSelfSignUpAvailable = config.getAuthenticationConfiguration().getEnableSelfSignup();
   }
 
+  private boolean isEmailServiceEnabled() {
```

Comment:
@Siddhanttimeline why are we writing same code multiple places, put it in Utils and use it